ape-config.yaml

```diff
@@ -5,12 +5,29 @@ plugins:
     version: 0.7.0
 
 dependencies:
-  - name: smartcontractkit
-    github: smartcontractkit/chainlink-brownie-contracts
-    version: 0.8.0
+  # - name: openzeppelin
+  #   github: eth-optimism/contracts
+  #   ref: 4.3.3
+
+  # - name: openzeppelin-contracts-upgradeable
+  #   gituhb: openzeppelin/contracts-upgradeable-4.7.3
+    
+  # - name: ethoptimism
+  #   github: eth-optimism/contracts
+  #   ref: 0.5.21
+
+  # - name: smartcontractkit
+  #   github: smartcontractkit/chainlink-brownie-contracts
+  #   version: 0.8.0
+
   - name: solmate
     github: transmissions11/solmate
     ref: "v6"
+    contracts_folder: src
+    config_override:
```

Comment:
so each dependency, you can override it's config as if there was an ape-config.yaml in that project.

typically what i do is clone each project independently, try running `ape compile` and seeing what all config i need to add in order for that to work. Then, copy that same config here into `config_override`...

**WARN**: I also to run `ape pm compile -f` to get rid of the faulty-compiled `solmate` that had no contracts.

but once i did all that, the `v6` directory was filled out.